src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final int ballCount = 100;
+
+    public String getRandomBall() {
+        int ballNumber = new Random().nextInt(ballCount);
```

Comment:
random.nextInt() returns 0 inclusively and 100 exclusively 
Random object creates every time you call getRandomColor. Think of the approach, where you create an object of Random only once for this class